src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py

```diff
@@ -1238,7 +1238,7 @@ def cn_op(lulc_array, soil_group_array):
         # if lulc_array value not in lulc_to_soil[soil_group_id]['lulc_values']
         # then numpy.digitize will not bin properly and cause an IndexError
         # during the reshaping call
-        lulc_unique = set(numpy.unique(lulc_array))
+        lulc_unique = set(i.item() for i in numpy.unique(lulc_array))
```

Comment:
I tried to look this up quickly, but I'm not sure I follow calling `item()` here...

Reply:
Ah, unless we're concerned with datatypes, I bet that's it.

Reply:
Yep, that's it exactly!  `i` would normally be a native numpy object (e.g. `numpy.float32`), so calling `.item()` gets the python representation of the object.